apps/omg_watcher_rpc/lib/web/response.ex

```diff
@@ -20,26 +20,21 @@ defmodule OMG.WatcherRPC.Web.Response do
   For the generic response, see `OMG.Utils.HttpRPC.Response`.
   """
 
-  @app :omg_watcher_rpc
+  alias OMG.WatcherRPC.Configuration
 
   @doc """
   Adds "version" and "service_name" to the response map.
   """
   @spec add_app_infos(map()) :: %{version: String.t(), service_name: String.t()}
   def add_app_infos(response) do
     response
-    |> Map.put(:version, version())
-    |> Map.put(:service_name, service_name())
-  end
-
-  defp version() do
-    OMG.Utils.HttpRPC.Response.version(@app)
-  end
-
-  defp service_name() do
-    case Application.get_env(@app, :api_mode) do
-      :watcher -> "watcher"
-      :watcher_info -> "watcher_info"
-    end
+    |> Map.put(:version, Configuration.version())
+    |> Map.put(
+      :service_name,
+      case Configuration.service_name() do
```

Comment:
nitpick. you can use `to_string`

Reply:
touche! or interpolation https://thepugautomatic.com/2016/01/elixir-string-interpolation-for-the-rubyist/
```
"#{Configuration.service_name()}"
```